crud/common/utils.lua

```diff
@@ -199,24 +199,23 @@ function utils.tarantool_supports_uuids()
     return enabled_tarantool_features.uuids
 end
 
-local function id_not_in_table(operations, id)
+local function exists_operations_id_map(operations)
```

Comment:
I would like this function name to start via `get_`.

Reply:
Also seems it's not `operations_id` it's field id.